### DISCUSSION TITLE
Random Sampler seed

### DISCUSSION THREAD

**Discussion by sfalkena** [torchgeo/torchgeo#2307] [Ideas]
Hi, I don't see it implemented currently, would there be any interest in allowing the `RandomGeoSampler` and `BatchGeoSampler` random seed to be set by allowing to pass a seed number to the sampler?

**Comment by adamjstewart** [comment]
PyTorch's builtin samplers all have an optional `generator` parameter. I would support this being a parameter in all of our code that uses RNGs, not just our samplers.
